package.json

```diff
@@ -18,6 +18,7 @@
     "coverage:upload": "nyc report --reporter=text-lcov | coveralls",
     "ern-debug": "node --nolazy --inspect-brk=5858 ern-local-cli/src/index.dev.js",
     "instrument": "lerna run instrument",
+    "prepare": "husky install",
```

Comment:
Can we leave this script out of package.json, and instead add instructions to README.md to run `yarn husky install` _for people who want husky set up locally_? I always prefer to run exactly what I want individually (which is why I've set an environment variable `HUSKY_SKIP_INSTALL` locally, but not sure if it is still recognized by updated husky).

Reply:
More context: https://github.com/typicode/husky/issues/468

Instead of using the `prepare` hook, maybe something like this would work too?

```
  "scripts": {
    "install-git-hooks": "npm install husky --no-save"
  },
```

Reply:
Yes, I agree that most contributors will be fine with the hooks being enabled by default (not many have complained so far ;)). That could be either because they find it useful and actually _want_ these hooks to be set up, or because they are _unaware_ of the scripts and don't mind the extra time it takes/consider it normal for this repo.

The point I'm trying to make:

1. Git is a purely **local** system (until commits are pushed to remote)
2. Verification of contributions and adherence to coding/formatting standards always needs to be done **remotely** anyway (or at least they should)

Exactly as you mentioned, local hooks can easily by skipped (`--no-verify`), while remote checks _cannot_ be skipped. Mixing the two _by default_ is not a good idea in my opinion (without an easy way to opt out).

Ultimately it's your decision, but hopefully you can give this another thought. In general it would be nice to _decrease_ the dependency on external tools and libraries over time.

A script/prompt that asks the first time it runs sounds like a good idea to me too..